torch/onnx/_internal/exporter.py

```diff
@@ -1147,10 +1147,10 @@ def dynamo_export(
         sarif_report_path = _DEFAULT_FAILED_EXPORT_SARIF_LOG_PATH
         resolved_export_options.diagnostic_context.dump(sarif_report_path)
         message = (
-            "Failed to export the model to ONNX. Generating SARIF report at {sarif_report_path}. "
-            "SARIF is a standard format for the output of static analysis tools. "
-            "SARIF log can be loaded in VS Code SARIF viewer extension, "
-            "or SARIF web viewer(https://microsoft.github.io/sarif-web-component/)."
+            f"Failed to export the model to ONNX. Generating SARIF report at '{sarif_report_path}'. "
+            f"SARIF is a standard format for the output of static analysis tools. "
+            f"SARIF logs can be loaded in VS Code SARIF viewer extension, "
+            f"or SARIF web viewer (https://microsoft.github.io/sarif-web-component/). "
```

Comment:
```suggestion
            "SARIF is a standard format for the output of static analysis tools. "
            "SARIF logs can be loaded in VS Code SARIF viewer extension, "
            "or SARIF web viewer (https://microsoft.github.io/sarif-web-component/). "
```
f-string is only needed when `{var}` is present. Keeping it unnecessarily might cause (minor) overhead

Reply:
I just thought it would be more readable since the strings are not missed by a single space. Either way works for me